include/deal.II/base/polynomials_rannacher_turek.h

```diff
@@ -63,6 +63,14 @@ class PolynomialsRannacherTurek
   */
   Tensor<2, dim> compute_grad_grad(const unsigned int i,
                                    const Point<dim> &p) const;
+  /** Hessian of gradient of basis function @p i at @p p.
+  */
+  Tensor<3, dim> compute_grad_hessian(const unsigned int i,
+                                      const Point<dim> &p) const;
+  /** Hessian of hessian of basis function @p i at @p p.
+  */
+  Tensor<4, dim> compute_grad_hessian_grad(const unsigned int i,
+                                           const Point<dim> &p) const;
```

Comment:
same here (and as appropriate in other classes): if you don't need these functions, don't implement them.